src/Form/EventRegistrationType.php

```diff
@@ -34,6 +35,13 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('acceptTerms', CheckboxType::class, [
                 'mapped' => false,
             ])
+            ->add('personalDataCollection', CheckboxType::class, [
```

Comment:
What about creat a new type: `AcceptPersonalDataCollectType` with this configuration? + add one template which will render the chebox element with label